modules/transfer-frontend/ecs.tf

```diff
@@ -38,6 +38,7 @@ data "template_file" "app" {
     draft_metadata_s3_bucket_name     = var.draft_metadata_s3_bucket_name
     block_automate_judgment_transfers = var.block_automate_judgment_transfers
     notification_sns_topic_arn        = var.notification_sns_topic_arn
+    file_checks_total_timout          = 480
```

Comment:
sorry, would it make sense to also name the parameter `file_checks_total_timeout_in_seconds` just so that it's clearer.